components/MezcalCard.js

```diff
@@ -0,0 +1,57 @@
+import Link from "next/link";
+import styled from "styled-components";
+import Image from "next/image";
+
+function MezcalCard({mezcal}) {
+  return (
+    <Link href="/article/[id]" as={`/article/${mezcal.id}`}>
+      <StyledCard>
+        <h3> {mezcal.brand} &rarr; </h3>
+        <p>{mezcal.type}</p>
+        <p>{mezcal.rating}</p>
+        <p>{mezcal.price}</p>
+
+        <Image
+          src={`${mezcal.image}`}
+          width="150"
+          height="150"
+          alt="an image"
```

Comment:
Your `alt`-attribute is used as an a11y attribute. You should think of a better description.